src/containers/Article.js

```diff
@@ -117,8 +128,9 @@ class Article extends Component {
     super(props, context)
 
     this._setArticleBounding = this._setArticleBounding.bind(this)
-    this._onScroll = this._onScroll.bind(this)
+    this._onScroll = _.throttle(this._onScroll, 200).bind(this)
```

Comment:
Since you use throttle to decrease scroll events, you can consider remove the `raf`.
I think `raf` will be redundant in the case.